src/tuneflow_py/plugins/isan_fusion/examples.py

```
@@ -0,0 +1,278 @@
+    IsanFusionPlugin.run(song, {
+        "instrument": "mor_lam",
+        "genre": "pop",
+        "mood": "happy",
+        "bpm": 128,
+        "key": "Em",
+        "duration_bars": 8,
+        "track_index": 2,
+        "insert_position": 0
+    })
+    
+    print(f"✅ Multi-track arrangement complete!")
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)